app/scripts/metamask-controller.js

```diff
@@ -1388,6 +1391,30 @@ export default class MetamaskController extends EventEmitter {
 
     ///: END:ONLY_INCLUDE_IF
 
+    ///: BEGIN:ONLY_INCLUDE_IF(notifications)
+    // Notification Controllers
+    this.userStorageController = new UserStorageController({
+      messenger: this.controllerMessenger.getRestricted({
+        name: 'UserStorageController',
+        allowedActions: [`${this.snapController.name}:handleRequest`],
+      }),
+      auth: {
```

Comment:
@Prithpal-Sooriya when will these methods be implemented? Can the ciide in this PR work without these methods?

Reply:
Yep this is implemented in the auth controller!
Once everything is merged I'll add the correct methods!

Reply:
I'll add some comments, make this explicit that this relies on a different PR.